com.github.matfantinel.reminduck.json

```diff
@@ -0,0 +1,72 @@
+{
+    "app-id": "com.github.matfantinel.reminduck",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno-19.08",
+    "command": "com.github.matfantinel.reminduck",
+    "finish-args": [
+       "--share=ipc",
+       "--share=network",
+       "--socket=x11",
+       "--socket=wayland",
+       "--talk-name=org.gtk.vfs.*",
+       "--talk-name=ca.desrt.dconf",
+       "--filesystem=home",
+       "--filesystem=xdg-run/dconf",
```

Comment:
You should remove dconf access. It's not needed anymore starting from FDO Sdk 19.08/GNOME Sdk 3.34